src/components/dashboard/SendTokens/BitcoinLikeBockchain/FormContainer.jsx

```diff
@@ -15,17 +14,16 @@ import {
   MODE_SIMPLE,
 } from 'components/constants'
 
-const mapStateToProps = (state, props) => ({
+export const mapStateToProps = (state, props) => ({
   token: state.get(DUCK_TOKENS).item(props.tokenSymbol)
 })
 
-function mapDispatchToProps (dispatch) {
+export function mapDispatchToProps (dispatch) {
```

Comment:
You shouldn't export mapDispatchToProps and mapStateToProps because it's a private implementation of connected component

Reply:
It doesnt look like React/Redux pattern

Reply:
Why not? Why is it private?

Reply:
Because these methods (mapDispatchToProps and mapStateToProps) are related only for the certain component. If you want to use props from another component you can pass it using spread operator (...), children or something more react way.
There is nothing bad with coping these methods as is especially when you have only one prop.

Reply:
Why you can't just put to the form props?

Reply:
Also, suggest move mapDispatchToProps from form to container. It's better